doc/source/releases/24.0_announce_user.rst

```diff
@@ -0,0 +1,196 @@
+
+Drag and drop:
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Keyboard navigation and selection:
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+Resource grids have been consolidated into tab views
+-----------------------------------------------------------
+
+Grids for Histories, Visualizations and Pages have been consolidated into tab views.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+UI for "relocating" a dataset to a new object store
+-----------------------------------------------------------
+
+Relocate a dataset to a new object store with ease using this new UI feature! The example below uses an instance with four object stores defined, but only three of them declaring the same "device" ID. Clicking on the dataset information and scrolling to storage details has a "Relocate" option if the dataset is "safe" to relocate and there are valid targets to relocate it to. The UI utilizes the same visual language used for describing attributes of the storage and exposing admin provided details. This example also shows what the buttons look like with quota enabled object stores and object stores without quota enabled.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="TODO" title="Galaxy 24.0 - " frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+A new comparison method for test output verification using images 
+-----------------------------------------------------------
+
+For tool developers, image_diff, a new comparison method for test output verification using images has been added. Unlike previously used comparison methods, image_diff is specifically tailored for single-channel and multi-channel image data (e.g. RGB). The difference of a pair of images is quantified as the pixel-wise distance between the images, for which different metrics can be used. A pair of images is considered to be equal in terms of the specified metric, if the distance between the images computed with respect to the metric is not above a given threshold. For more details, see `the original pull request <https://github.com/galaxyproject/galaxy/pull/17556>`__).
```

Comment:
I'm not sure about including this here, tool devs aren't the target audience of the last 20 iterations of this file (but maybe that's changing?), this could go in the full `announce.rst` file instead perhaps?